lib/modeling/Collection.js

```diff
@@ -1,15 +1,8 @@
 "use strict";
 
 var Backbone = require("../application/Backbone");
-var Syncing = require("./Syncing");
 
-var Collection = Backbone.Collection.extend({
-
-    sync: Syncing.sync
-
-});
-
-module.exports = Collection;
+module.exports = Backbone.Collection;
```

Comment:
I assume you're just leaving this and `Brisket.Model` around to minimize breakage?